make/Coverage.gmk

```diff
@@ -34,21 +34,28 @@ else
   JCOV_INPUT_IMAGE_DIR := $(JDK_IMAGE_DIR)
 endif
 
+JCOV_SUPPORT_DIR := $(SUPPORT_OUTPUTDIR)/jcov
+
 #moving instrumented jdk image in and out of jcov_temp because of CODETOOLS-7902299
-JCOV_TEMP := $(SUPPORT_OUTPUTDIR)/jcov_temp
+JCOV_TEMP := $(JCOV_SUPPORT_DIR)/temp
+
+ifneq ($(JCOV_MODULES), )
```

Comment:
Is this redundant conditional done on purpose?

Reply:
You mean as in a foreach on an empty variable will yield nothing? Yes, I thought that made to code clearer, but if you think otherwise I can remove it.

Reply:
No, it's fine, I just wanted to know if it was intentional for code clarity.